src/reducers/gotPlayer/counts.js

```diff
@@ -27,9 +27,22 @@ export const getPlayerCounts = {
     }
     return state.app.gotPlayer.counts.byId[id];
   },
-  getError: (state, id) => getPlayerCounts.getPlayerCountsById(state, id).error,
-  getLoading: (state, id) => getPlayerCounts.getPlayerCountsById(state, id).loading,
-  isLoaded: (state, id) => getPlayerCounts.getPlayerCountsById(state, id).loaded,
+  getOnlyData: (state, id) => {
```

Comment:
I had to refactor my reducerFactory method I was using for counts (which is a weird case since it is multiple tables in one request) to also work for histograms. Part of that meant changing the data structure of the reducer. This selector strips out the loading information from counts which means that I can still iterate over the object keys which are different counts tables to generate those tables.